packages/airview-compliance-ui/src/features/progress-bar/progress-bar.js

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import LinearProgress from "@mui/material/LinearProgress";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
```

Comment:
For consistency, import as a named import: 

`import { LinearProgress, Typography, Box } from "@mui/material";`